bower.json

```diff
@@ -18,7 +18,8 @@
     "lazysizes": "^2.0.4",
     "moment": "^2.18.1",
     "three.js": "threejs#*",
-    "mobile-detect": "hgoebl/mobile-detect.js#^1.3.6"
+    "mobile-detect": "hgoebl/mobile-detect.js#^1.3.6",
```

Comment:
Looks like this is surplus to requirement?

Reply:
ah yeah, should be removed